src/lib/components/sidebar/Sidebar.component.js

```diff
@@ -176,16 +176,19 @@ function Sidebar({
             <Button
               size="larger"
               variant="backgroundLevel1"
-              icon={<i className="fas fa-chevron-left" />}
-              onClick={() => onToggleClick()}
+              icon={<i className="fas fa-bars" />}
+              onClick={() => {
+                onToggleClick();
+                setHovered(false);
```

Comment:
onToggleClick may unmount the sidebar component and in those case this will raise a warning because you are trying to update the state of an unmounted component